### PR TITLE
Creation of Alton Towers, Chessington and Thorpe Park Brands

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do? (please provide any background)
- Creates 2 new brands in our brand repo, one for Thorpe Park and one for Alton Towers
- Modify's the exiting Chessington brand to work a little better.

#### What tests does this PR have?
None

#### How can this be tested?
This is linked to another PR - 
Ensure you are on this branch of brands
Clone `pattren-library` and switch to branch `alton`
`npm link ../brands/`
`npm start` on the pattern library
go to `pages` and select `shortbreaks payment`
Now switch brands, play with your screen size and test.

#### Screenshots / Screencast
screen_shot_2016-02-05_at_17 20 11_720
screen_shot_2016-02-05_at_17 22 08_720

#### What gif best describes how you feel about this work?
YEP

---

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

#### Reviewers

**Review 1**
- [x] :+1:

**Review 2** \*
- [x] :+1:

**Review 3** _(optional)_
- [ ] :+1:

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru

### CODE REVIEW

File: brands/alton-towers/package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@holidayextras/brand-altontowers",
+  "description": "A styling package for the Alton Towers brand",
```
Comment (elliottcrush) [inline brands/alton-towers/package.json]: can we add `"version": "1.0.0",` in here

File: brands/thorpe-park/package.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@holidayextras/brand-thorpepark",
+  "description": "A styling package for the Thorpe Park brand",
```
Comment (elliottcrush) [inline brands/thorpe-park/package.json]: can we add `"version": "1.0.0",` in here

**Comment by elliottcrush** [comment]
:one:

**Comment by elliottcrush** [comment]
:+1:

**Comment by nickine** [comment]
:+1:

**Pull request merged**